backend/blog/api/queries.py

```diff
@@ -41,9 +43,10 @@ def resolve_used_tags(root, info, **kwargs):
         tags = [obj.tag for obj in TaggedItem.objects.select_related('tag').all()]
         return list(set(tags))
 
-    def resolve_posts(root, info, **kwargs):
+    def resolve_paginated_posts(root, info, **kwargs):
         category_slug = kwargs.get('category_slug', None)
         tag_slug = kwargs.get('tag_slug', None)
+        page_nr = kwargs.get('page_nr', None)
```

Comment:
ich würde mich hier auf ein naming für die aktuelle page einigen. (page_nr vs page vs active-page im FE/BE)